config/paths-osx.conf

```diff
@@ -25,3 +25,5 @@ syslog_authpriv = /var/log/secure.log
 
 #syslog_local0 =
 
+# Default Mac OS log location for syslog output.
```

Comment:
Hmm, since the unit tests appear to not load it I guess there is no point to it then.